runtimes/datascience/ubi9-python-3.9/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM quay.io/opendatahub/workbench-images:runtime-minimal-ubi9-python-3.9-pr-58
```

Comment:
This should be replaced by ARG, or it needs to rebase the PR

Reply:
Good catch
fixed it.